src/Rewrite_Command.php

```diff
@@ -27,6 +27,7 @@
  *
  * @package wp-cli
  */
+// phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedClassFound -- for back compat this class shouldn't be renamed.
```

Comment:
This might have an impact on how the preceding docblock is attributed to the class.

Please use an exception regex in the ruleset for class names, like we did here: https://github.com/wp-cli/cron-command/pull/46/files#diff-6758ef1f416bd888bd1a5cad8f97cc2bR56